serverless/tensorflow/faster_rcnn_inception_v2_coco/nuclio/function-gpu.yaml

```diff
@@ -114,7 +114,8 @@ spec:
         - kind: RUN
           value: ln -s faster_rcnn_inception_v2_coco_2018_01_28 faster_rcnn
         - kind: RUN
-          value: pip install pillow pyyaml
+          value: pip install pillow<10 pyyaml
```

Comment:
You need to quote the argument with the `<` sign, otherwise it will be treated as a redirection.